src/components/filters/helpers.js

```diff
@@ -1,6 +1,6 @@
 import moment from 'moment';
 
-export const oldestDate = new Date(1950, 1, 1);
+export const oldestDate = new Date(1970, 1, 1);
```

Comment:
btw.: ![Screenshot 2023-06-27 at 00 44 47](https://github.com/RedHatInsights/insights-inventory-frontend/assets/7757/4da89646-296d-434c-a6e7-8916eda00e47)

Reply:
You can pass Date a value when constructing. From the MDN docs[0]:

> An integer value representing the [timestamp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#the_epoch_timestamps_and_invalid_date) (the number of milliseconds since midnight at the beginning of January 1, 1970, UTC — a.k.a. the [epoch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#the_epoch_timestamps_and_invalid_date)).


So, O milliseconds since January 1, 1970, gives January 1, 1970. :D

[0] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date

Reply:
That doesnt seem like new. Date(0), this is the oldestDate const I updated

Reply:
Oh thats actually just a huge coincidence haha

Reply:
It's not a coincidence: https://en.wikipedia.org/wiki/Unix_time

But yeah, it does make sense to have it explicit. Maybe. :)

Reply:
That's probably also why the backend has that as the minimum date. :)